pointerevents/pointerevent_compatibility_events_on_prevent_default-manual.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Pointer Event: Prevent compatibility mouse events</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
+    <link rel="author" title="Google" href="http://www.google.com "/>
+    <meta name="assert" content="When a pointerdown is canceled, selected compatibilty mouse events are not fired until a pointerup."/>
+    <link rel="stylesheet" type="text/css" href="pointerevent_styles.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script type="text/javascript" src="pointerevent_support.js"></script>
+    <script type="text/javascript">
+      var test_pointerEvent = async_test("Prevent compat mouse events");
+      add_completion_callback(end_of_test);
+
+      var detected_pointertypes = {};
+      var event_log = [];
+
+      function end_of_test() {
+          document.getElementById("event-log").innerHTML =
+              "Logged the following events: " +
+              (event_log.length ? event_log.join(", ") : "(none)");
+          showPointerTypes();
+      }
+
+      function end_of_interaction() {
+          test(function () {
+              assert_equals(event_log.join(", "),
+                        "pointerdown@target0, " +
+                        "pointerup@target0, " +
+                        "pointerdown@target1, mousedown@target1, " +
+                        "pointerup@target1, mouseup@target1");
```

Comment:
Done, ptal.

Also split the test into two cases: click/tap (for any pointer type) and drag (only for mouse).